networktest.go

```diff
@@ -245,34 +245,38 @@ type networkHarness struct {
 	watchRequests chan *watchRequest
 }
 
-// newNetworkHarness creates a new network test harness given an already
-// running instance of btcd's rpctest harness. Any extra command line flags
-// which should be passed to create lnd instance should be formatted properly
-// in the lndArgs slice (--arg=value).
-// TODO(roasbeef): add option to use golang's build library to a binary of the
-// current repo. This'll save developers from having to manually `go install`
-// within the repo each time before changes.
-func newNetworkHarness(lndArgs []string) (*networkHarness, error) {
+/*
+Creates a new network test harness.
+*/
+func newNetworkHarness() (*networkHarness, error) {
 	return &networkHarness{
 		activeNodes:   make(map[int]*lightningNode),
 		seenTxns:      make(chan wire.ShaHash),
 		watchRequests: make(chan *watchRequest),
 	}, nil
 }
 
-func (n *networkHarness) InitializeSeedNodes(r *rpctest.Harness) error {
+ /*
```

Comment:
The comment for this method should be altered to match the style described within the [Commentary section of Effective Go](https://golang.org/doc/effective_go.html#commentary). So use single line comments rather than a block comment, and start the comment with the name of the method (`InitializeSeedNodes`). 

As an aside: if you're new to Go (whcih I think is the case), then [Effective Go](https://golang.org/doc/effective_go.html) is a great resource in general to come up to speed with some of the currently considered best practices and idioms within golang.